actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -497,6 +508,17 @@ def expiry_options(options)
           end
         end
 
+        def cookie_metadata(options)
+          if request.use_cookies_with_metadata
+            metadata = expiry_options(options)
```

Comment:
Moving this into here is a bit too late, isn't it? We've already shipped writing the expiry information directly in the cookie in Rails 5.2 haven't we?

Reply:
@kaspth 

Andrew White, had asked us to give an option to turn off both the metadata, since a lot of applications might be at various stages of upgradation.

Also, since we were writing cookies with the expiry information, in Rails 5.2.0 (Stable) Rafael Franca had given an option to turn off metadata since the new format makes it impossible to read   on 5.0.

Reply:
@assain where's @rafaelfranca's option at, I don't see it here?